inputFiles/thermalMultiphaseFlow/co2_thermal_2d.xml

```diff
@@ -187,27 +192,28 @@
 
     <FieldSpecification
       name="sourcePressure"
-      setNames="{ source }"     
-      objectPath="ElementRegions/region/cb"
+      logLevel="1"
```

Comment:
Yes, I prefer to keep the log levels to 1 in some integrated tests, because newcomers may modify these existing tests to construct their own xml files, and I think it is good to start with the log levels to 1 to avoid mistakes.